src/core/eligibility.ts

```diff
@@ -58,6 +60,9 @@ function canBeExecuted(stage: Stage, mv: MaterializedView): boolean {
     // eslint-disable-next-line @typescript-eslint/ban-ts-comment
     // @ts-ignore -  Change by good deepEqual
```

Comment:
Isn't it the case ?

Reply:
No, I would like to improve this deepEqual in the future, with a not-homemade solution